lib/resque/scheduler.rb

```diff
@@ -23,11 +23,11 @@ module Scheduler
     public
 
     class << self
+      attr_accessor :logger
```

Comment:
Continue using `attr_writer :logger` since you are already defining the method for the reader below